pkg/sentry/kernel/task_start.go

```diff
@@ -176,7 +176,7 @@ func (ts *TaskSet) newTask(ctx context.Context, cfg *TaskConfig) (*Task, error)
 	t.netns = cfg.NetworkNamespace
 	t.creds.Store(cfg.Credentials)
 	t.endStopCond.L = &t.tg.signalHandlers.mu
-	t.seccomp.Store((*taskSeccomp)(nil))
+	t.seccomp.Store(nil)
```

Comment:
This line can be deleted; it was only needed for `atomic.Value` so that the case in `t.seccomp.Load().(*taskSeccomp)` could be unconditional.

Reply:
Good catch, thanks!